src/team.py

```diff
@@ -0,0 +1,10 @@
+class Team:
+    def __init__(self, name, players, matches=[], flag=None, coach=''):
```

Comment:
Perhaps you should remove the **matches** parameter, this information will be stored in the _schedule_. Make a schedule method for getting national team matches. Something like _get_matches(team_name)_